doctest/doctest.h

```diff
@@ -2057,9 +1975,17 @@ int registerReporter(const char* name, int priority) {
     DOCTEST_GLOBAL_NO_WARNINGS_END() typedef int DOCTEST_ANONYMOUS(_DOCTEST_ANON_FOR_SEMICOLON_)
 
 // for logging
-#define DOCTEST_INFO(x)                                                                            \
-    doctest::detail::ContextScope DOCTEST_ANONYMOUS(_DOCTEST_CAPTURE_)(                            \
-            doctest::detail::ContextBuilder() << x)
+#define DOCTEST_INFO(expression)                                                                   \
+    DOCTEST_INFO_IMPL(DOCTEST_ANONYMOUS(_DOCTEST_CAPTURE_), expression)
+
+#define DOCTEST_INFO_IMPL(lambda_name, expression)                                                 \
+    auto lambda_name = [&](std::ostream* s) {                                                      \
+        doctest::detail::MessageBuilder mb(__FILE__, __LINE__, doctest::assertType::is_warn);      \
```

Comment:
```mb``` can conflict with a local variable captured with ```[&]``` - that's why I proposed to have the name of the message builder as an argument to the macro as well - like the lambda

Reply:
Ahh, I was wondering why that was the case. I'll add it back then.